code/obj/item/device/pda2/modules.dm

```diff
@@ -321,3 +322,64 @@
 		var/obj/item/device/pda_module/alert/J = the_item
 		if (J.host)
 			J.send_alert()
+
+/obj/item/device/pda_module/flashlight/yell_medic
+	name = "medic siren module"
+	desc = "A PDA module that lets you announce your presence as medical personnel to your surroundings."
+	icon_state = "pdamod_medic"
+	abilities = list(/obj/ability_button/pda_flashlight_toggle, /obj/ability_button/yell_medic)
+	var/datum/light/MedicLight
```

Comment:
Consider changing this to a simple light, as robust lights look very laggy when attached to a moving object (and the player is always going to be moving if they're pressing this)